app/org/sagebionetworks/bridge/json/DateUtils.java

```diff
@@ -3,21 +3,37 @@
 import org.apache.commons.lang3.StringUtils;
 import org.joda.time.DateTime;
 import org.joda.time.DateTimeZone;
+import org.joda.time.LocalDate;
 import org.joda.time.Period;
 import org.joda.time.format.DateTimeFormatter;
 import org.joda.time.format.ISODateTimeFormat;
 import org.joda.time.format.ISOPeriodFormat;
 import org.joda.time.format.PeriodFormatter;
 
+import org.sagebionetworks.bridge.BridgeConstants;
+
 public final class DateUtils {
 
     private static final DateTimeFormatter dateFmt = ISODateTimeFormat.date();
     private static final DateTimeFormatter dateTimeFmt = ISODateTimeFormat.dateTime();
     private static final PeriodFormatter periodFmt = ISOPeriodFormat.standard();
 
-    private static DateTime getDateTime() {
+    /** Returns current time as a Joda DateTime object. */
+    public static DateTime getCurrentDateTime() {
         return DateTime.now(DateTimeZone.UTC);
     }
+
+    /** Returns current calendar date as a Joda LocalDate object, using UTC as the reference time zone. */
+    public static LocalDate getCurrentCalendarDateInUtc() {
+        return LocalDate.now(DateTimeZone.UTC);
+    }
+
+    /**
+     * Returns current calendar date as a Joda LocalDate object, using Pacific local time as the reference time zone.
+     */
+    public static LocalDate getCurrentCalendarDateInLocalTime() {
+        return LocalDate.now(BridgeConstants.LOCAL_TIME_ZONE);
```

Comment:
I'm a little confused by this because once you set a time zone, it's no longer a local date, it's a date date. Or is that not true because there's no time component?